### ISSUE TITLE
torch.max not working

### ISSUE THREAD

**Issue by weinixuehao** [alibaba/TinyNeuralNetwork#268]
I implement a custom PReLU with the following code.

```
class PReLU2(nn.Module):
    def __init__(self) -> None:
        super(PReLU2, self).__init__()
        self.weight = nn.Parameter(torch.tensor(0.25))

    def forward(self, x):
        return torch.max(torch.zeros_like(x), x) + self.weight * torch.min(torch.zeros_like(x), x)
```

Failed to convert with the following exception.
```
Exception has occurred: RuntimeError       (note: full exception trace is shown but execution is paused at: _run_module_as_main)
shape '[1]' is invalid for input of size 2621440
  File "/Users/mac/workspace/.venv/lib/python3.8/site-packages/tinynn/converter/operators/torch/base.py", line 559, in handle_reduce
    self.output_tensors[0] = self.output_tensors[0].view(1)
  File "/Users/mac/workspace/.venv/lib/python3.8/site-packages/tinynn/converter/operators/torch/aten.py", line 3299, in parse
    self.handle_reduce(tfl.ReduceMaxOperator, args, graph_converter, False)
  File "/Users/mac/workspace/.venv/lib/python3.8/site-packages/tinynn/converter/base.py", line 443, in init_operations
    converter.parse(node, attrs, args, self.common_graph)
  File "/Users/mac/workspace/.venv/lib/python3.8/site-packages/tinynn/converter/base.py", line 480, in convert
    self.init_operations()
  File "/Users/mac/workspace/mobile_model_converter.py", line 16, in convertToTflite
    converter.convert()
  File "/Users/mac/workspace/mobile_model_converter.py", line 82, in <module>
    convertToTflite(deshadowNet, 'deshadow')
  File "/Users/mac/.pyenv/versions/3.8.5/lib/python3.8/runpy.py", line 87, in _run_code
    exec(code, run_globals)
  File "/Users/mac/.pyenv/versions/3.8.5/lib/python3.8/runpy.py", line 194, in _run_module_as_main (Current frame)
```

The root of the problem is this torch.max() operator.
![image](https://github.com/alibaba/TinyNeuralNetwork/assets/17869361/1845ea15-a17b-4c64-bd50-796e1614717d)

**Comment by peterjc123** [comment]
@weinixuehao Yes, it is because we only support the reduction logic of `torch.max/min` before (e.g. torch.max(x, dim=-1)). We will add the support for `torch.maximum/mininum` for transforming elementwise ops first.  And then we will fix this issue.

**Comment by peterjc123** [comment]
@weinixuehao With https://github.com/alibaba/TinyNeuralNetwork/pull/269, it should work if you replace `torch.max/min` with `torch.maximum/minimum`.
